apps/docs/src/content/docs/file-system-operations.mdx

```diff
@@ -28,7 +28,7 @@ print(f"Is directory: {file.is_dir}")
 print(f"Size: {file.size}")
 print(f"Modified: {file.mod_time}")
 
-```
+````
```

Comment:
Seems that you formatted the repo. This will be broken now. 
Docs should skip formatting for now until we resolve the formatter